resources/js/Components/Collections/Nfts/NftHeader/NftActions.tsx

```diff
@@ -0,0 +1,145 @@
+import { Clipboard } from "@/Components/Clipboard";
+import { NetworkIcon } from "@/Components/Networks/NetworkIcon";
+import { Report } from "@/Components/Report";
+import { Tooltip } from "@/Components/Tooltip";
+import { useToasts } from "@/Hooks/useToasts";
+import { ExplorerChains } from "@/Utils/Explorer";
+
+interface Properties {
+    alreadyReported?: boolean;
+    className?: string;
+    nft: App.Data.Nfts.NftData;
+    reportAvailableIn?: string | null;
+    reportReasons?: Record<string, string>;
+}
+
+export const NftActions = ({
+    alreadyReported = false,
+    className,
+    nft,
+    reportAvailableIn = null,
+    reportReasons,
+}: Properties): JSX.Element => {
+    const { t } = useTranslation();
+    const { showToast } = useToasts();
+
+    const [isRefreshing, setIsRefreshing] = useState(false);
+
+    const { large: largeImage, original, originalRaw } = nft.images;
+    const originalNftImage = original ?? originalRaw ?? largeImage;
+
+    const handleRefresh = async (): Promise<void> => {
+        setIsRefreshing(true);
+
+        await axios.post<{ success: boolean }>(
+            route("nft.refresh", {
+                collection: nft.collection.slug,
+                nft: nft.tokenNumber,
+            }),
+        );
+
+        showToast({
+            message: t("common.refreshing_metadata"),
+            isExpanded: true,
+        });
+    };
+
+    const getTooltipText = (chainId: App.Enums.Chains): string => {
+        switch (chainId) {
+            case ExplorerChains.EthereumMainnet:
+            case ExplorerChains.EthereumTestnet:
+                return t("common.view_nft_on_etherscan").toString();
+            default:
+                return t("common.view_nft_on_polygonscan").toString();
+        }
+    };
+
+    const getChainLink = (chainId: App.Enums.Chains, collectionAddress: string, tokenNumber: string): string => {
+        switch (chainId) {
+            case ExplorerChains.EthereumMainnet:
+            case ExplorerChains.EthereumTestnet:
+                return t("urls.explorers.etherscan.token_transactions", {
+                    address: tokenNumber,
+                    token: collectionAddress,
+                });
+            case ExplorerChains.PolygonTestnet:
+                return t("urls.explorers.mumbai.token_transactions", {
+                    address: tokenNumber,
+                    token: collectionAddress,
+                });
+            default:
+                return t("urls.explorers.polygonscan.token_transactions", {
+                    address: tokenNumber,
+                    token: collectionAddress,
+                });
+        }
+    };
+
+    const handleClick = (): Window | null =>
+        window.open(getChainLink(nft.collection.chainId, nft.collection.address, nft.tokenNumber), "_blank");
+
+    return (
+        <div
+            className={cn("inline-flex gap-2 rounded-full", className)}
+            data-testid={"NftActions__container"}
```

Comment:
all these icons need to be the color of the background

design:
<img width="539" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/35610748/8f26bea5-7cc1-45d6-9877-1ee41f47e19a">

implementation:
<img width="238" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/35610748/8ffed656-abf4-4d60-9162-1f24de3cf639">